BUILD.bazel

```diff
@@ -57,7 +57,10 @@ cc_library(
     srcs = ["src/Angle.cc"],
     hdrs = ["include/gz/math/Angle.hh"],
     includes = ["include"],
-    deps = [":Helpers"],
+    deps = [
+        ":Helpers",
+        ":Config",
```

Comment:
I think the buildifier test is failing because these are supposed to be alphabetical.

You can fix with `bazel run //:buildifier.fix`

Reply:
Done.